### DISCUSSION TITLE
Delaying workflow executions triggered by sub-process node till the next executable time defined by timing management

### DISCUSSION THREAD

**Discussion by rebeccasoren** [apache/dolphinscheduler#12461] [Q&A]
I have a workflow-A with a sub-process node that triggers workflow-B. workflow-B also has a timing setting that is "online". 
However, workflow-B is triggered as soon as the sub-process node is executed, disregarding the timing setting. 
<img width="360" alt="2022-10-20 13_13_29-Dolphin Scheduler Admin" src="https://user-images.githubusercontent.com/51917800/196888425-bb416aed-9a0e-427e-89f6-e03ceda7d84b.png">

Can execution of workflow-B's be delayed until the next executable time specified in its timing management page?

**Comment by DarkAssassinator** [comment]
hi @rebeccasoren, so u mean that the delay time do not take affect for condtion task?

**Reply by gzqcason** [reply]
I guess he was confused why the workflow-B execute immediately instead of running at next scheduling timestamp (for workflow-B), not the delay time option doesn't work in specific tasks.
